index.d.ts

```diff
@@ -0,0 +1,23 @@
+export type LocationUnion = LatitudeLongitude | LatLng | LatLon;
+
+interface LatitudeLongitude {
+    latitude: number;
+    longitude: number;
+}
+
+interface LatLng {
+    lat: number;
+    lng: number;
+}
+
+interface LatLon {
+    lat: number;
+    lon: number;
+}
+
+/**
+ * Return the Haversine distance in meters
+ * @param a first location
+ * @param b second location
+ */
+export default function haversineDistance(a: LocationUnion, b: LocationUnion): number;
```

Comment:
space between `distance` and `(` probably

Reply:
I am using the one of, if not "the" default lint for ts but we can add a space if needed…
```suggestion
export default function haversineDistance (a: LocationUnion, b: LocationUnion): number;
```